Parent-Commands.md

```diff
@@ -11,22 +11,35 @@ Key things to remember from the main page:
 ___
 
 ### Index
-*  [`info`](#lp-usergroup-usergroup-parent-info)
+*  [`info`](#lp-usergroup-usergroup-parent-info-page-sorting-mode)
 *  [`set` \<group\> [context...]](#lp-usergroup-usergroup-parent-set-group-context)
 *  [`add` \<group\> [context...]](#lp-usergroup-usergroup-parent-add-group-context)
 *  [`remove` \<group\> [context...]](#lp-usergroup-usergroup-parent-remove-group-context)
-*  [`settrack` \<track\> \<group\> [context...]](#lp-usergroup-usergroup-parent-settrack-track-group-context)
+*  [`settrack` \<track\> \<index | group\> [context...]](#lp-usergroup-usergroup-parent-settrack-track-indexgroup-context)
 *  [`addtemp` \<group\> \<duration\> [temporary modifier] [context...]](#lp-usergroup-usergroup-parent-addtemp-group-duration-temporary-modifier-context)
 *  [`removetemp` \<group\> [context...]](#lp-usergroup-usergroup-parent-removetemp-group-context)
 *  [`clear` [context...]](#lp-usergroup-usergroup-parent-clear-context)
 *  [`cleartrack` \<track\> [context...]](#lp-usergroup-usergroup-parent-cleartrack-track-context)
 *  [`switchprimarygroup` \<group\>](#lp-user-user-parent-switchprimarygroup-group)
 
 ___
-#### `/lp user/group <user|group> parent info`  
+#### `/lp user/group <user|group> parent info [page] [sorting mode]`  
 **Permission**: luckperms.user.parent.info or luckperms.group.parent.info  
+**Arguments**:  
+* `[page]` - the page number to view
+* `[sorting mode]` - how the results will be sorted
+  
 Displays a list of a user/group's parent groups. (groups they inherit from)
 
+The "sorting mode" argument allows you to specify how the list will be sorted. You can pick between 4 different options.
+
+| Sorting mode             | Description                                                              |
```

Comment:
(style consistency suggestion) Make the descriptions start with capital letters like everywhere else